webapp/client/src/pages/PersonAHasDisabilityPage.js

```diff
@@ -0,0 +1,80 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { Trans, useTranslation } from "react-i18next";
+import FormHeader from "../components/FormHeader";
+import FormFieldRadio from "../components/FormFieldRadio";
+import StepForm from "../components/StepForm";
+import StepHeaderButtons from "../components/StepHeaderButtons";
+import Details from "../components/Details";
+import { fieldPropType } from "../lib/propTypes";
+
+export default function PersonAHasDisabilityPage({
+  form,
+  fields,
+  prevUrl,
+  stepHeader,
+  numOfUsers,
+}) {
+  const { t } = useTranslation();
+
+  const tbold = function (key) {
+    return <Trans t={t} i18nKey={key} components={{ bold: <b /> }} />;
+  };
+
+  let headerIntro = tbold("lotse.hasDisability.intro_1");
+
+  if (numOfUsers > 1) {
+    headerIntro = tbold("lotse.hasDisability.intro_2");
+  }
+
+  return (
+    <>
+      <StepHeaderButtons url={prevUrl} />
+      <FormHeader title={stepHeader.title} intro={headerIntro} />
+      <StepForm {...form}>
+        <Details
+          title={t("lotse.hasDisability.details.title")}
+          detailsId="person_a_has_disability"
+        >
+          {{
+            paragraphs: [tbold("lotse.hasDisability.details.text")],
+          }}
+        </Details>
+        <FormFieldRadio
+          fieldId="person_a_has_disability"
+          fieldName="person_a_has_disability"
+          options={[
+            {
+              value: "yes",
+              displayName: t("fields.switch.Yes"),
+            },
+            {
+              value: "no",
+              displayName: t("fields.switch.No"),
+            },
+          ]}
+          value={fields.personA_hasDisability.value}
+          errors={fields.personA_hasDisability.errors}
+          required
+        />
```

Comment:
If details are before a field, so basically part of that field, you can add the details prop to the call of the RadioField. I tried it out and somehow it only seems to work if there is a label set. I don't think it should be like that but maybe I'm wrong. 
Long story short: If the Details semantically belongs to the page, leave it as is - if it belongs to the field I'd try to make the effort to make the RadioField display it in its own label.

Reply:
We talked about it. Let's just leave it like that.